frontend/src/core/services/PocketUserService.js

```diff
@@ -194,6 +194,51 @@ class PocketUserService extends PocketBaseService {
     return `******${lastLetters}${emailProvider}`;
   }
 
+    /**
+   * Retrieve security questions from server.
+   *
+   * @return {Promise|Promise<{success:boolean, [data]: *}>}
+   * @async
+   */
+  async getSecurityQuestions() {
+    return axios
+      .get(`${Configurations.backendUrl}/api/security_questions`)
+      .then(response => {
+        if (response.status === 200)
+          return { success: true, data: response.data };
+      })
+      .catch(err => {
+        return { sucess: false, data: err };
+      });
+  }
+
+  /**
+   * Send security questions to server.
+   *
+   * @param {string} email Email of user
+   * @param {Array.<{question:string, answer:string}>} questions Questions and
+   * answers of user
+   * @return {Promise|Promise<{success:boolean, [data]: *}>}
+   * @async
+   */
+  async sendSecurityQuestions(email, questions) {
```

Comment:
Move this part to https://github.com/pokt-network/pocket-dashboard/blob/master/frontend/src/core/services/PocketSecurityQuestionsService.js, this is already a file for these data.